CONTRIBUTING.md

```diff
@@ -95,6 +95,23 @@ cd OkapiLib
 git submodule update --init --recursive
 ```
 
+To build the Okapi library:
```

Comment:
I just realized you wrote this as well. 

The `mkdir template` command should not be needed. 
>  Which will result in the okapilib.a file being created in the `bin` directory.

Yes, but what you care about is that it will result in `okapilib@version.zip` created in the root directory, which you can then import into the PROS conductor and then apply it to your projects.

Also, might want to add that the commands need to be run on linux, with gcc, make, cmake, and git installed.